### PR TITLE
[PWD-2528] Finalize My Account Security

### PR DESCRIPTION
**Pull request by gangsararyop**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Nitip review API sekalian guys

https://github.com/PurwadhikaDev/purwadhika-api-new/pull/1095

### CODE REVIEW

File: src/views/account/components/SecuritySection.tsx [hunk 1]
```diff
@@ -24,51 +37,54 @@ export const SecuritySection = () => {
...
+            )}
+            helperText={
+              touched.confirmationPassword && errors.confirmationPassword
+            }
+          />
+        </Stack>
       </SectionWrapper>
-      <Button
-        size="small"
+      <LoadingButton
```
Comment (FahemHakikiKhaya) [inline src/views/account/components/SecuritySection.tsx:79]: sizenya lupa kebawa mas gang

Comment (gangsararyop) [reply src/views/account/components/SecuritySection.tsx:79]: pake default size nya hem, size small itu salah terlalu kecil

File: src/components/PasswordInput.tsx [hunk 2]
```diff
@@ -1,21 +1,18 @@
 import { Visibility, VisibilityOff } from '@mui/icons-material'
-import { IconButton, InputAdornment, TextField } from '@mui/material'
+import { InputAdornment, TextField } from '@mui/material'
 import { FC, useState } from 'react'
 
 interface PasswordInputProps {
```
Comment (hiljaker) [inline src/components/PasswordInput.tsx:5]:
```suggestion
interface PasswordInputProps extends ComponentProps<typeof TextField> {
```

**Review by hiljaker — APPROVED** [review]
𝖘ikattt om gang

**Review by kuntiarso — APPROVED** [review]
🔆🔆

**Pull request merged**